### ISSUE TITLE
The template is 404

### ISSUE THREAD

**Issue by vmachacek** [MicrosoftDocs/architecture-center#3714]
the link for auto deploy is 404 
https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/umbraco-cms-webapp-redis-cache/azuredeploy.json

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 6a2bfab3-1734-dc0d-74c1-319da054d591
* Version Independent ID: 46b135ba-49ef-7bc2-66cd-066959fca7e7
* Content: [Scalable Umbraco CMS web app - Azure Solution Ideas](https://learn.microsoft.com/en-us/azure/architecture/solution-ideas/articles/medium-umbraco-web-app)
* Content Source: [docs/solution-ideas/articles/medium-umbraco-web-app.yml](https://github.com/microsoftdocs/architecture-center/blob/main/docs/solution-ideas/articles/medium-umbraco-web-app.yml)
* Service: **architecture-center**
* Sub-service: **solution-idea**
* GitHub Login: @EdPrice-MSFT
* Microsoft Alias: **architectures**

**Comment by Naveenommi-MSFT** [comment]
@vmachacek 
Thanks for your feedback! We will investigate and update as appropriate.

**Comment by MughundhanRaveendran-MSFT** [comment]
@vmachacek , I have reviewed the links in this doc : https://learn.microsoft.com/en-us/azure/architecture/solution-ideas/articles/medium-umbraco-web-app and I dont see the broken link that you have mentioned. Could you please point me to the exact section in the above mentioned documentation? Thanks

**Comment by MughundhanRaveendran-MSFT** [comment]
@vmachacek , Did you get a chance to look into my previous comment? Please let me know

**Comment by vmachacek** [comment]
![image](https://user-images.githubusercontent.com/2987885/196554226-21318fe8-c045-48bb-bfdb-53a0514f096a.png)
here

**Comment by MughundhanRaveendran-MSFT** [comment]
Yes. I can see the that the template is not available. I am assigning it to author to fix this

![image](https://user-images.githubusercontent.com/56299140/196595885-186aacae-34ef-4305-945b-eed07767d585.png)

**Comment by ckittel** [comment]
Thanks for pointing this out @vmachacek.  I've pushed an update to the link that should be live within about 24 hours.  The new link is <https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fapplication-workloads%2Fumbraco%2Fumbraco-cms-webapp-redis-cache%2Fazuredeploy.json> so you don't need to wait any longer :)
